getting-started/bare-metal/about.md

```diff
@@ -1,56 +1,64 @@
 ---
 title: About non-cluster hosts
-description: Secure hosts not in a cluster by installing Calico with networking and/or networking policy enabled.
+description: Secure hosts not in a cluster by installing Calico with networking and/or networking policy.
 canonical_url: '/getting-started/bare-metal/about'
 ---
 
 ### Big picture
 
-Secure hosts not in a cluster by installing {{site.prodname}} with networking and/or networking policy enabled.
+Secure hosts by installing {{site.prodname}} for networking and/or networking policy.
 
 ### Value
 
-Not all hosts in your environment run virtualized workloads (i.e. containers managed by Kubernetes or OpenShift, or VMs managed by OpenStack). There may be physical machines or legacy applications that you cannot move into an orchestrated cluster that need to communicate securely with workloads in your cluster. Whether you have a thousand machines or ten, {{site.prodname}} lets you enforce policy on them using the same robust {{site.prodname}} network policy that is used for workloads. 
+Not all hosts in your environment run pods/virtualized workloads (i.e. containers managed by Kubernetes or OpenShift, or VMs managed by OpenStack). You may also have physical machines or legacy applications that need to securely communicate with pods, which cannot be move into an orchestrated cluster. You can secure these hosts (called **non-cluster hosts**) by installing {{site.prodname}} networking and network policy, or only network policy. The advantage of using {{site.prodname}} to protect the host are:
 
+- You get all of the {{site.prodname}} networking options: BPG, or overlay with VXLAN, IP-in-IP
+- You can use the same {{site.prodname}} policy configuration as you do with pods  
+    For example, you can write a cluster-wide policy for hosts that is immediately applied to every host. Use one tool to protect both hosts and pods. For details on how to restrict traffic to/from hosts using {{site.prodname}} network policy see, [Protect hosts]({{site.baseurl}}/security/protect-hosts)
 
 ### Concepts
 
-#### Workloads and Hosts
```

Comment:
Don't we want a sub-heading here?

e.g.,

```
#### Non-cluster host
```

Reply:
Yes, good catch.